### PR TITLE
[test/plugin] immigrate test servicecomb scenario

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583

### CODE REVIEW

**Comment by arugal** [comment]
Hi, @dmsolr @kezhenxu94 test passred. Please review.

File: test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# lists your version here
+
+0.1.0
+0.2.0
+0.3.0
+0.4.0
+0.5.0
```
Comment (wu-sheng) [inline test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list]: Should have more releases today, could you recheck?

Comment (arugal) [reply test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list]: No more versions were found in the [central repo](https://mvnrepository.com/artifact/io.servicecomb)

Comment (arugal) [reply test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list]: I'll add a new scenario for 1.x

Comment (wu-sheng) [reply test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list]: I think they move to Apache after that :) I don't know whether the plugin works for that or not. cc @WillemJiang Anything related to SkyWalking changed?

Comment (wu-sheng) [reply test/plugin/scenarios/servicecomb-0.x-scenario/support-version.list]: https://github.com/apache/servicecomb-java-chassis/releases

File: test/plugin/scenarios/servicecomb-1.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/servicecomb/provider/CodeFirstJaxrsHelloImpl.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.testcase.servicecomb.provider;
+
+
+import org.apache.servicecomb.provider.pojo.RpcSchema;
```
Comment (wu-sheng) [inline test/plugin/scenarios/servicecomb-1.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/servicecomb/provider/CodeFirstJaxrsHelloImpl.java]: The package name has been changed, I think we need a new plugin?

Comment (arugal) [reply test/plugin/scenarios/servicecomb-1.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/servicecomb/provider/CodeFirstJaxrsHelloImpl.java]: Already provided [servicecomb-plugin](https://github.com/apache/skywalking/tree/master/apm-sniffer/apm-sdk-plugin/servicecomb-plugin)

Comment (wu-sheng) [reply test/plugin/scenarios/servicecomb-1.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/servicecomb/provider/CodeFirstJaxrsHelloImpl.java]: I see.

**Pull request merged**